CONTRIBUTING.md

```diff
@@ -1087,27 +1094,26 @@ for the simple commands we use for this.
 
 To run clang-tidy locally, follow these steps:
 
-1. Install clang-tidy. First, check if you already have clang-tidy by simply
-writing `clang-tidy` in your terminal. If you don't yet have clang-tidy, you
-should be able to install it easily with your package manager, e.g. by writing
-`apt-get install clang-tidy` on Ubuntu. See https://apt.llvm.org for details on
-how to install the latest version. Note that newer versions of clang-tidy will
-have more checks than older versions. In our CI, we run clang-tidy-6.0.
-
-2. Use our driver script to run clang-tidy over any changes relative to some
-   git revision (you may want to replace `HEAD~1` with `HEAD` to pick up
-   uncommitted changes). Changes are picked up based on a `git diff` with the
-   given revision:
-  ```bash
-  git diff HEAD~1 > pr.diff
-  python tools/linter/clang_tidy --paths torch/csrc --diff-file "pr.diff"
-  ```
+1. Install clang-tidy.
```

Comment:
We use the Makefile below and these are encoded in `make setup_lint`, should we suggest that here instead of running the steps manually? It installs some other packages for the other linters but devs probably want those anyways